keps/sig-node/4800-cpumanager-split-l3cache/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: Split L3 Cache Toplogy Awareness in CPU Manager
+kep-number: 4800
+authors:
+  - "@sphrasavath"
+  - "@wongchar"
+  - "@ajcaldelas"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable 
+creation-date: 2024-08-23
+reviewers:
+  - "@ffromani"
+  - "@kannon92"
+approvers:
+  - "@mrunalp" 
+  - "@sig-node-leads"
+
+see-also: 
+  - keps/sig-node/2625-cpumanager-policies-thread-placement"
+
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.32"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.32"
+  beta: "v1.33"
+  stable: "v1.34"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: "CPUManagerPolicyOptions"
```

Comment:
Need clarity on this... why should we not rely on CPUManagerPolicyOptions. This would be where the user can enable the uncore-cache-alignment feature

Reply:
Because you have no way to roll back this feature without disabling all cpu policies.

Besides this feature should be GA soon so this feature gate will be marked as true forever. If you want this alpha this I think you need a separate feature gate.

Reply:
We can and should use `CPUManagerPolicy{Alpha,Beta}Options`. The per-policy option use a different flow (ack'd by sig-arch) to avoid proliferation of feature gates and managing in (hopefully) a bit more efficiently their lifecycle.
xref: https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L110C1-L117C83

Reply:
I fully agree, it's pretty clear in xref

Reply:
Thank you! Yea let's mention that somewhere in the doc.